secure_message/repository/database.py

```diff
@@ -157,6 +170,37 @@ def serialize(self):
         return data
 
 
+class Actors(db.Model):
+    """Label Assignment table model"""
+    __tablename__ = "actors"
+
+    id = Column("id", Integer(), primary_key=True)
+    msg_id = Column('msg_id', String(constants.MAX_MSG_ID_LEN + 1), ForeignKey('secure_message.msg_id'), index=True)
+    from_actor = Column('from_actor', String(constants.MAX_STATUS_ACTOR_LEN + 1))
+    to_actor = Column('to_actor', String(constants.MAX_STATUS_ACTOR_LEN + 1))
```

Comment:
Are these always `+1`? Would it be advantageous to create the constant with the extra in the first place?

Reply:
The idea here is that the constant values are used for validation purposes and the +1 defines the column size so that  the column size can always cope with any value that passes validation even if we get a \0 added at some point .

Reply:
Is this premature future proofing?

Reply:
Not future proofing .Its easier to add this defensively at the beginning knowing that validation of all external entities is done at sizes less than the column capacity than to never know if you get a column overflow if it was caused by data passed in or some muck up in code. By doing this I know that if I ever get an overflow error and I can proove that the validation is to less than that size , any overflow must be introduced by our code not the data passed in. To me it makes things an awful lot cleaner and easier to understand . I would argue its good practice